osu.Game/Overlays/NowPlayingOverlay.cs

```diff
@@ -234,9 +234,9 @@ protected override void Update()
                 pendingBeatmapSwitch = null;
             }
 
-            var track = beatmap.Value?.TrackLoaded ?? false ? beatmap.Value.Track : null;
+            var track = musicController.CurrentTrack;
```

Comment:
.

Reply:
i don't think we want to revert this one, right..?

Reply:
This one can stay as is.